#coefficient for single planes;

uint32 index
float64 a #ax+by+cz+d=0 is plane's equation
float64 b 
float64 c
float64 d